scripts/vlm/neva_finetune.py

```diff
@@ -143,7 +143,9 @@ def main(args):
                 image_processor=image_processor,
                 multimodal_sample_config=config,
                 packed_sequence=args.use_packed_sequence,
-                packed_sequence_size=decoder_seq_length,
+                # leave some space for perf padding, otherwise after packing and padding,
+                # it will go beyond max seq len, then it will need a truncation.
+                packed_sequence_size=int(decoder_seq_length * 0.9),
```

Comment:
I think a better way for this might be to take seq lengths into account while packing? Like here:

https://github.com/NVIDIA/NeMo/blob/abhi/llava_next_sp/nemo/collections/vlm/neva/data/sequence_packing.py#L124

Reply:
that's hard, because it contains multiple padded seqs. so wont easy to do a accurate calc, can only estimate